CONTRIBUTING.md

```diff
@@ -187,12 +187,14 @@ The purpose is to separate problem from possible solutions.
 
 **Bug fixes:** If you’re only fixing a small bug, it’s fine to submit a pull request right away but we highly recommend to file an issue detailing what you’re fixing. This is helpful in case we don’t accept that specific fix but want to keep track of the issue. Please keep in mind that the project maintainers reserve the rights to accept or reject incoming PRs, so it is better to separate the issue and the code to fix it from each other. In some cases, project maintainers may request you to create a separate issue from PR before proceeding.
 
-**Refactor:** For small refactors, it can be a standalone PR itself detailing what you are refactoring and why. If there are concerns, project maintainers may request you to create a `#SIP` for the PR before proceeding.
+**Refactor:** For small refactors, it can be a standalone PR itself detailing what you PRare refactoring and why. If there are concerns, project maintainers may request you to create a `#SIP` for the PR before proceeding.
```

Comment:
```suggestion
**Refactor:** For small refactors, it can be a standalone PR itself detailing what you are refactoring and why. If there are concerns, project maintainers may request you to create a `#SIP` for the PR before proceeding.
```